tests/sample/bindmonitor_bpf2bpf.c

```diff
@@ -23,23 +23,37 @@ bind_action_t
 BindMonitor_Callee(uint64_t* pid);
```

Comment:
do we need to regenerate the "generated" files for this sample program?

Reply:
Done